.github/workflows/ci.yml

```diff
@@ -89,43 +89,33 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]
-        containerd: [v1.6.21, v1.7.1]
+        containerd: [1.6.21, 1.7.1]
 
     steps:
       - name: Checkout extensions
         uses: actions/checkout@v4
 
+      - name: Download containerd archive
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
FYI: passing the default token is required when using in the workflow https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows